src/routes/guides/building-a-data-source/+page.svelte

```diff
@@ -50,319 +46,75 @@
-        "languages": ["fi", "sv"],
-        "area": 338455,
-    },
-    "NR": {
-        "code": "NR",
-        "name": "Nauru",
-        "capital": None,
-        "languages": ["na", "en"],
-        "area": 21,
-    },
+  "temperature": 17.3,
+  "humidity": 72,
+  "pressure": 1007,
+  "wind_speed": 2.1,
+  "wind_direction": 220,
+  "precipitation": 120,
+  "visibility": 320
 }
-
-
-async def get_data(country: str) -> Dict[str, Any]:
-    """
-    Get the country data.
-
-    This would in practice fetch the data from your underlying data source,
-    such as a database, sensor or another API.
-
-    :param country: Two-letter country code.
-    :return: The data for the country.
-    :raise KeyError: If no data is found for the country.
-    """
-    return data[country.upper()]
-`}
-  </Code>
-  <p>
-    This contains just a hardcoded dictionary of different values and a simple wrapper to fetch
-    them. In practice you would want to change this to actually somehow fetch or generate the
-    necessary data. Note that you might also need to install some Python packages for connecting to
-    and querying your database or other systems. The original implementation required an
-    <em> API_KEY</em>
-    and an <em>API_ENDPOINT</em>, which our data source does not require, so we can remove those
-    from <em>settings.py</em>. Those can however be handy examples for your own implementation.
-  </p>
-  <SectionTitle title="Adding the route" />
-  <p>
-    Let's open the file <em>app/routers/dataproduct.py</em>. We start with emptying it, so we get
-    rid of the old route. Let's add our new route like this:
-  </p>
-  <Code lang={python}>
-    {`
-from fastapi import APIRouter
-from fastapi.exceptions import HTTPException
-
-from app.datasource import get_data
-from app.models import BasicCountryInfoRequest, BasicCountryInfoResponse
-
-router = APIRouter()
-
-
-@router.post(
-    "/test/ioxio-dataspace-guides/Country/BasicInfo",
-    summary="Basic Country Info",
-    description="Data Product for basic country info",
-    response_model=BasicCountryInfoResponse,
-)
-async def data_product(params: BasicCountryInfoRequest):
-    try:
-        data = await get_data(params.code)
-    except KeyError:
-        raise HTTPException(404, "Country not found")
-
-    return BasicCountryInfoResponse(**data)
 `}
   </Code>
   <p>
-    We define the route for the path <em>/test/ioxio-dataspace-guides/Country/BasicInfo</em>, which
-    matches the path for our definition and the path defined in the OpenAPI spec file. We also
-    define which pydantic models we use for the request and for the response, as well as define some
-    metadata for the route. We use our <em>get_data</em> function to retrieve the data, raising a 404
-    exception in case the country is not found.
+    You can build this using your own preferred tools, or take our
+    <A href="https://github.com/ioxio-dataspace/data-source-guide-example">integration</A> as a refference.
```

Comment:
I don't think I'd want to call it an integration; it's not integrating into any other system, I'd rather call it an implementation. Also there's a typo in `refference` (shoudl have only one `f`).

```suggestion
    You can build this using your own preferred tools, or have a look at our
    <A href="https://github.com/ioxio-dataspace/data-source-guide-example">simple example implementation</A>.
```